### DISCUSSION TITLE
docker cann't work

### DISCUSSION THREAD

**Discussion by powellsz** [OpnForm/OpnForm#324] [Q&A]
![pic](https://github.com/JhumanJ/OpnForm/assets/19164258/d35a4e76-ca80-4998-8ba6-3417d6b86196)

**Comment by powellsz** [comment]
`docker run --name opnform -v $PWD/my-opnform-data:/persist -p 8080:80 jhumanj/opnform:latest`

**Reply by powellsz** [reply]
@JhumanJ

**Reply by JhumanJ** [reply]
Please provide docker logs, and the output of the console. What's the API request response?

**Reply by powellsz** [reply]
no response as server's url is wrong.

**Reply by powellsz** [reply]
![1](https://github.com/JhumanJ/OpnForm/assets/19164258/c101b9fb-a5fe-49c8-b38d-6adf49bd1508)

**Reply by powellsz** [reply]
Listening on http://[::]:3000
5.180.78.52 - - [20/Feb/2024:11:17:22 +0000] "GET /login HTTP/1.1" 200 26907 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36"

**Reply by powellsz** [reply]
@JhumanJ

**Reply by JhumanJ** [reply]
URL looks right to me. Anything in the browser console?

**Reply by powellsz** [reply]
@JhumanJ i deployed it to a public domain ip , api url should have the same server ip as the 124.xxx.xxx.xxx ,not localhost.

**Comment by JhumanJ** [comment]
Closing this as we've made a lot of progress on the docker setup since.
